scoreboard_player_runes.py

```diff
@@ -0,0 +1,41 @@
+import re
+import json
+
+from mwcleric.fandom_client import FandomClient
+from mwcleric.auth_credentials import AuthCredentials
+from mwcleric.template_modifier import TemplateModifierBase
+from mwparserfromhell.nodes import Template
+from mwparserfromhell.nodes.extras.parameter import Parameter
+
+from runes_reforged_parser import RunesReforgedParser
+
+rune_parser = RunesReforgedParser()
+
+class TemplateModifier(TemplateModifierBase):
+    def update_template(self, template: Template):
+        for param in template.params:
```

Comment:
a maybe easier way to do this is:

```python
if not template.has('runes'):
    return
 param = template.get('runes')
```

There's no way to `.get()` a template from a wikitext parsed tree, but you can just get a param from a template, we don't have to iterate through all params

Reply:
Ah, fixed